TIBERIANDAWN/HOUSE.CPP

```diff
@@ -3806,6 +3806,9 @@ TechnoTypeClass const * HouseClass::Suggest_New_Object(RTTIType objecttype) cons
 				**	A computer controlled house will try to build a replacement
 				**	harvester if possible. Never replace harvesters if the game
 				**	is in easy mode.
+				**		CFE Note: Vanilla bugfix - Difficulty is CORRECTLY set to check !DIFF_EASY here, but not in two other spots that also control the same harvester replacement logic.
+				**		Jump to the others by searching the below phrase:
+				**			CFE Harvester Build Fix
 				*/
 				if (!(GameToPlay == GAME_NORMAL && PlayerPtr->Difficulty == DIFF_EASY) && !IsHuman && (ActiveBScan & STRUCTF_REFINERY) && !(UScan & UNITF_HARVESTER)) {
```

Comment:
@Nyerguds - Pinging you so there's no surprises or anything since you had originally replied on the Steam page for the mod, however I think something kinda got lost in translation when trying to communicate the issue through imgur screencaps lol.

Would appreciate your input on this, as I'm FAIRLY confident this is infact the intended behavior that the other iterations should be set to.